building/build-debs/homeworld-keysystem/inner-build.sh

```diff
@@ -26,5 +26,8 @@ export GOPATH="${GODIR}:$(pwd)"
 go build src/keyserver/main/keyserver.go
 go build src/keygateway/main/keygateway.go
 go build src/keyclient/main/keyclient.go
+go build src/keygen.go
```

Comment:
More scripts at the top level which should probably be in their own `main` packages.

Reply:
Ah, yeah.